k8s/configMap-res.yaml

```diff
@@ -56,22 +56,22 @@ data:
     Rules:
       - Id: helios
         FrontendRegexp: ^[^\.]+\.k8s\.wikia\.net/helios
-        Sampling: 1.0
-      - Id: helios-legacy
+        Sampling: 0.5
+      - Id: helios
         FrontendRegexp: ^services\.wikia\.com/auth
-        Sampling: 1.0
-      - Id: helios-dev-legacy
-        FrontendRegexp: ^services\.wikia-dev\.(pl|us)/auth
-        Sampling: 1.0
-      - Id: helios-dev-k8s
-        FrontendRegexp: ^services-k8s\.wikia-dev\.(pl|us)/helios
-        Sampling: 1.0
-      - Id: helios-dev-k8s
-        FrontendRegexp: ^prod\.res\.k8s\.wikia\.net/helios
-        Sampling: 1.0
+        Sampling: 0.5
       - Id: services-all
         FrontendRegexp: ^services\.wikia\.com
-        Sampling: 1.0
+        Sampling: 0.5
+      - Id: services-all
```

Comment:
can we bind it to different id? we don't really want to base traefik apdex alerts based on internal traffic

Reply:
Sure. Let me update that

Reply:
@tmszdmsk a249063